### PR TITLE
mentor_obligations.md: Update faq

### PR DESCRIPTION
**Pull request by ShivamSinghania**
Add mentor roles
Closes https://github.com/coala/projects/issues/503

### CODE REVIEW

File: _faq/mentor_obligations.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 question: "What are my obligations as a mentor during the GSoC?"
 ---
+coala has two mentors for every project:
+1. Primary mentor - responsible for the administrative tasks of the project.
```
Comment (jayvdb) [inline _faq/mentor_obligations.md]: blank line above a list.

File: _faq/mentor_obligations.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
...
+2. Secondary mentor - has little to no responsibility, but provides backup and code review for aspects of the project which are relevant to their expertise.
```
Comment (jayvdb) [inline _faq/mentor_obligations.md]: cut lines at 80cpl

File: _faq/mentor_obligations.md [hunk 3]
```diff
@@ -10,3 +14,5 @@ Participate in weekly meetings with your student.
 * Review your students pull requests timely.
 * Get in contact with the admins if any even remotely
 potential problems arise.
+
+If you are interested in mentoring, please contact one of the admins indicating the reason your OSS work is suitable to be included as a project to be mentored under the coala banner. List of admins is [here](http://projects.coala.io/#/mentors).
```
Comment (jayvdb) [inline _faq/mentor_obligations.md]: "OSS work" -> "open source project"

"banner" -> "administrative umbrella".

new paragraph for "List of .."

File: _faq/mentor_obligations.md [hunk 4]
```diff
@@ -1,6 +1,12 @@
 ---
 question: "What are my obligations as a mentor during the GSoC?"
 ---
+coala has two mentors for every project:
+
+1. Primary mentor - responsible for the administrative tasks of the project.
+2. Secondary mentor - has little to no responsibility, but provides backup and
+code review for aspects of the project which are relevant to their expertise.
```
Comment (jayvdb) [inline _faq/mentor_obligations.md]: indent according to normal markdown numbered list style.

Comment (ShivamSinghania) [reply _faq/mentor_obligations.md]: Sorry, it's unclear. What do you want me to do here ? I have used ordered list according to https://guides.github.com/features/mastering-markdown/#syntax
What do indent ?

**Comment by jayvdb** [comment]
Ack https://github.com/coala/projects/commit/9d09883f24af5be1a0298f5ec4d6177be67fc88b

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**